from app.core.exceptions.base import InternalServerError


class ServerError(InternalServerError):
    pass
